config/routes.rb

```diff
@@ -6,12 +6,16 @@
   resources :posts
   resources :comments, only: [:create]
   resources :likes
+  resources :friendships, only: [:create, :destroy, :index]
+  post '/delete_pending_request', to: 'friendships#delete_pending_request'
```

Comment:
- [ ] This could be the same as destroy, generated earlier with `resources :friendships, only: [:create, :destroy, :index]`